scripts: dts: Remove deprecated extract_dts_includes.py script

galak opened the pull request:
We now use EDTS and gen_defines.py to generate DTS defines.  We
deprecated the old script and defines several releases ago, so lets now
remove them.

Signed-off-by: Kumar Gala <kumar.gala@linaro.org>

galak committed:
scripts: dts: Remove deprecated extract_dts_includes.py script

We now use EDTS and gen_defines.py to generate DTS defines.  We
deprecated the old script and defines several releases ago, so lets now
remove them.

Signed-off-by: Kumar Gala <kumar.gala@linaro.org>

ulfalizer reviewed (changes requested):
`scripts/dts/devicetree.py` is also part of the old scripts.

galak commented:
> `scripts/dts/devicetree.py` is also part of the old scripts.

Thanks, now also removed ;).

galak committed:
boards: lpcxpresso55s69: Fix use of deprecated macro

In board pinmux.c ifdef of the form DT_GPIO_KEYS_SWx_GPIO_CONTROLLER
should be DT_GPIO_KEYS_SWx_GPIOS_CONTROLLER.

Signed-off-by: Kumar Gala <kumar.gala@linaro.org>
